### PR TITLE
Simple Hugo static site for the documentation

### PR DESCRIPTION
**Pull request by zregvart**
This configures the Hugo static site generator for publishing of the
documentation. The simple `hugo-book` theme suits our purposes for now.

### CODE REVIEW

**Comment by zregvart** [comment]
Preview can be seen [here](https://zregvart.github.io/ec-cli/)

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "docs/themes/hugo-book"]
+	path = docs/themes/hugo-book
+	url = ../../alex-shpak/hugo-book
```
Comment (lcarva) [inline .gitmodules:3]: Any reason not to use `github.com/alex-shpak/hugo-book`?

File: internal/documentation/documentation.go [hunk 2]
```diff
@@ -17,59 +17,72 @@
...
-	cmd.RootCmd.DisableAutoGenTag = true
+	}()
 
-	// Create the target paths
-	for _, p := range paths {
-		if err := os.MkdirAll(p, os.ModePerm); err != nil {
-			errs = multierror.Append(errs, err)
-			continue
+	// Markdown
+	if *markdown != "" {
```
Comment (lcarva) [inline internal/documentation/documentation.go:50]: Could this be a null pointer?

**Review by lcarva — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (zregvart) [reply .gitmodules]: Probably something my git client optimized, switched to full URL...

File: internal/documentation/documentation.go (hunk 2 above)
Comment (zregvart) [reply internal/documentation/documentation.go:50]: [Nope](https://pkg.go.dev/flag#String)

**Pull request merged**